server/models/migrations/next/migrate.sql

```diff
@@ -79,3 +79,15 @@ ALTER TABLE `service` DROP COLUMN `id`;
 
 
 ALTER TABLE `service` ADD PRIMARY KEY (`uuid`);
+  ('collectionCapacity', 'REPORT.COLLECTION_CAPACITY.TITLE');
+
+/**
+@author: lomamech:
+@date: 2020-07-09
+*/
+
+ALTER TABLE `province` DROP INDEX `province_1`;
+
+ALTER TABLE `sector` DROP INDEX `sector_1`;
```

Comment:
These needs to drop the key, not just the index.  I am still running into errors: 

![image](https://user-images.githubusercontent.com/896472/87141985-c2ac1b80-c29b-11ea-9828-df37e0962bba.png)

Reply:
With the Vanga database, I was able to mix all the occurrences of DR Congo, and we can also see that there are even several Kwilu Province
![image](https://user-images.githubusercontent.com/8035588/87145885-fd18b700-c2a1-11ea-996c-c0e24c2e5fdb.png)

Reply:
@jniles  Try to check in the structure of the province table, with the error message it seems to me that the index province_1 is not delete
![image](https://user-images.githubusercontent.com/8035588/87146231-8af4a200-c2a2-11ea-94b6-4242e879f400.png)